client/components/navbar.jsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 
+function hashRoute() {
+  window.location.hash = '#';
+}
+
 function Nav(props) {
   return (
     <div className="container">
       <div className="navbar row">
         <div className="left-nav">
-          <h1 className='logo'>FireFuze</h1>
+          <h1 className='logo' onClick={hashRoute} >FireFuze</h1>
```

Comment:
If your click target only changes the hash, it should be an anchor element. You may need to style it a bit.
```jsx
<h1 className='logo'><a href="#">FireFuze</a></h1>
```